organization/search_indexes.py

```diff
@@ -0,0 +1,205 @@
+from haystack import indexes
+
+from search.base_search_indexes import SearchEntity
+from sfm_pc.templatetags.countries import country_name
+
+
+class OrganizationIndex(SearchEntity, indexes.Indexable):
+
+    '''
```

Comment:
what is this comment doing for us?

Reply:
Ah, I included these as a reference for what needs to go into each index. I think it's safe to remove!